### PR TITLE
Issue 987/survey status chip

### LINKED ISSUE #987: Survey status chip
## Description
A user should be able to know the status of the survey

## Relevant Job Stories
When a survey is no longer relevant, I want to close the survey, so that no more people can respond to it

## Prerequisites
Don't start until the pages have been created in #982 

## Requirements
- [x] Add status chip in survey header 
- [x] Add status chips objects : open, active, scheduled, draft and closed.

## Possible implementations
Mimic the patterns used in call assignments (`CallAssignmentStatusChip.tsx`).

## Design specifications
Figma --> https://www.figma.com/file/EjLcpjKWgNizCb5PRBF51P/Zetkin-3?node-id=2517%3A36483&t=Zm38Y7wFONKOsZfv-0

Open survey status
chipppp

List of survey status (Active is not relevant for surveys)
chipppppppps

### PR DESCRIPTION
**Pull request by kaulfield23**
## Description
It shows survey status in chip. If user make survey unpublished, then chip changes. 

## Screenshots
image
image

## Changes

* Adds tests for surveyDataModel.
* Adds key to organizeSurvey/en.yml
* Adds path to organizeSurvey in localeScope
* Adds SurveyStatusChip

## Notes to reviewer
I solved merge conflicts twice with rebase. I checked several times to make sure nothing is missing. But let me know if something is missing!

## Related issues
Resolves #987

### CODE REVIEW

File: src/features/surveys/models/SurveyDataModel.spec.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      const mockData = mockStoreData('2022-03-23', '2022-04-01');
+      const store = createStore(mockData);
+
+      const apiClient = instance(mockClient);
+      const env = new Environment(store, apiClient, instance(mockRouter));
+      const model = new SurveyDataModel(env, 1, 1);
+      expect(model.state).toBe(SurveyState.UNPUBLISHED);
+    });
+
+    it('is PUBLISHED when publish date is past and expireDate is in the future', () => {
```
Comment (olleolleolle) [inline src/features/surveys/models/SurveyDataModel.spec.ts:96]: Minor: Perhaps `expireDate` OR `expire date` (as in the previous test `it` description).

**Review by olleolleolle — COMMENTED** [review]
Looks good, very regular!

File: src/features/surveys/models/SurveyDataModel.spec.ts (hunk 1 above)
Comment (kaulfield23) [reply src/features/surveys/models/SurveyDataModel.spec.ts:97]: Thanks! I will fix it now!

File: src/features/surveys/models/SurveyDataModel.spec.ts [hunk 2]
```diff
@@ -24,6 +24,7 @@ function mockItem<DataType extends { id: number }>(data: DataType) {
   item.loaded = new Date().toISOString();
   return item;
 }
+
```
Comment (olleolleolle) [inline src/features/surveys/models/SurveyDataModel.spec.ts:27]: ✨ _Nicer!_

**Review by olleolleolle — COMMENTED** [review]
That got nicer.

**Review by richardolsson — APPROVED** [review]
Looks good to me!

**Pull request merged**